### DISCUSSION TITLE
Add option to edit path to apps in split tunnel settings

### DISCUSSION THREAD

**Discussion by RileyXX** [mullvad/mullvadvpn-app#9090] [Feature requests & Ideas]
### Feature request

It'd be nice if there were an option to edit the path to apps in the split tunnel window rather than just showing the app name. Some other apps like Nord have this feature but not Mullvad for some reason. I need to edit the path of one of the Chivalry 2 apps in the screenshot below but since I can't edit them or see which one is which, I have to remove them both and readd them with is both annoying and time consuming.

<img width="240" alt="04-18-2023_277" src="https://user-images.githubusercontent.com/49823202/232906444-fa9eeba9-1165-4c51-95cf-28f1b188c64a.png">

## Issue report

Operating system: Windows 10
App version: 2023.3

**Comment by LinuxOnTheDesktop** [comment]
More: the app positively needs to _show_ the paths. Otherwise this type of confusion results:

![image](https://github.com/user-attachments/assets/28eddaed-d1de-45e7-977e-f23a94188c2d).

Further: one wonders just _when_ the settings get applied.

(And adding one or other or both of those apps to the 'split tunnel' seems not to work, but at least so doing no longer completely severs my Internet connection. See [this meta-bug report](https://github.com/mullvad/mullvadvpn-app/issues/6459) for these sorts of problems.)
